ambari-server/src/main/resources/stack-hooks/before-ANY/scripts/params.py

```diff
@@ -207,7 +207,8 @@ def is_secure_port(port):
 if dfs_ha_nameservices is None:
   dfs_ha_nameservices = default('/configurations/hdfs-site/dfs.nameservices', None)
```

Comment:
It seems like the `dfs_ha` stuff should be included within the hdfs-site check (for consistency) but they seem to _protect_ themselves.